### DISCUSSION TITLE
Is this project still being worked on?

### DISCUSSION THREAD

**Discussion by RefineryX** [jonaswinkler/paperless-ng#1533] [General]
I installed paperless before the holidays and have it set up beautifully. There’s a couple of  pull requests but have been approved but not yet released. I see that nothing has been worked on since September. Is it still an active project or am I missing something?

**Comment by auanasgheps** [comment]
That's very sad. The project looks amazing, but has been quiet since a while. The list of PRs is quite large! I hope it will be updated again,

**Comment by mskg** [comment]
I asked myself the same question. But I saw a comment from the repo owner yesterday. It seems to be worked on? 
@jonaswinkler  ;)

**Reply by auanasgheps** [reply]
I hope that's a good sign :)

**Comment by as0bit** [comment]
Is there something new? I think the project is really great and it would be really sad if it was no longer maintained. It would be great if you could give us a little info @jonaswinkler

**Comment by sytone** [comment]
Its hard to manage a project like this when it is not your job, also harder when you are the only maintainer and it works for you and you do not need anything else. Not sure if anyone has offered to help with maintenance and management of the discussion, issues and PRs. That in itself is a big job assuming @jonaswinkler wants to take on having other people in here doing changes as well. 

Sometimes projects go quite because life as well.

**Reply by RefineryX** [reply]
Totally get that and all part of open source management. I am hoping that @jonaswinkler considers building out a selected team of maintainers to assist with issues and PRs given that it is a such a critical project for many folks. Having it siloed to one person can be concerning, whilst we completely appreciate Jonas's time and effort maintaining the project and taking over from the old depository. Having responsive folks available can also be beneficial to avoid the burnout of one person and to also contribute when the unexpected happens such as if/when vulnerabilities are identified etc.

**Comment by as0bit** [comment]
I know the problem with the lack of free time and projects very well. Unfortunately I can't help technically but I would like to donate money to @jonaswinkler . That doesn't solve the time problem either, but it might motivate you.
I'd rather give money to good open source projects than bad mainstream products.
And this project saves a lot of time in my life which I can then use for my projects. :)

**Comment by azapater** [comment]
I completely understand that this kind of "on my free time" projects are tricky to maintain, mostly having a main job and as some of you say, a life itself. But that's the beauty of a successful open source project: usually there are enough people out there to help and take off some of that pressure from the main developer. 

I use paperless for my personal use and the little invoicing I do sometimes for a few customers, so I'm more than willing to help with the project if I can. I've never worked with django and very little with angular (I'm more of a PHP + Vue kind of developer) but as I call myself: "I'm an apprentice of everything and a master of nothing" so it's a perfect excuse to learn django and/or angular in a junior level. If someone else more senior with these languages wants to supervise my work and other people's, it would be great to keep the project afloat. Obviously, as long as @jonaswinkler is happy with it.

**Comment by mskg** [comment]
I'm currently using it and did some fixes/enhancements for myself, too. 
Can help - no problem. Did professional work with Angular, Python.

**Comment by sytone** [comment]
I have looked at making some smaller changes but managed to work outside of it like the script to make folders for all meta data and then symlink. Would like to have more event hooks and looked at implementation but holding off on forking to see what @jonaswinkler wants to do about pull requests and external contributions.

**Comment by beedaddy** [comment]
Before I feed paperless-ng with all my documents this would also be an important question for me....

**Comment by cleaverm** [comment]
The discussion on #1599 / #1632 has been pretty active and is looking promising 🤞
